Description: Avoid stripping binary. dh_We want to generate the dbgsym package too.
Author: Marcos Fouces <marcos@debian.org>
---
This patch header follows DEP-3: https://dep.debian.net/deps/dep3/
--- a/Makefile
+++ b/Makefile
@@ -25,8 +25,8 @@
 
 all: ${OBJECTS}
 	cc $(OBJECTS) -o $(OUT) $(LDFLAGS) $(LIBS)
-	strip $(OUT)
+#	strip $(OUT)
 
 clean:
 	rm -rf $(OBJECTS) $(OUT)
-	
\ No newline at end of file
+	
